.travis.yml

```diff
@@ -3,5 +3,5 @@ script:
  - npm run build
  - npm test
 node_js:
+  - "stable"
   - "8"
-  - "6"
```

Comment:
Node 6 was already unsupported, because `async`/`await` was being used.

Reply:
Can you revert this please 👍